pallets/loans/src/benchmarking.rs

```diff
@@ -102,6 +110,7 @@ fn transfer_initial_balance<
         true,
     )
     .ok();
+    T::Assets::mint_into(HKO, &caller, INITIAL_AMOUNT.into()).unwrap();
```

Comment:
Refer to this: https://github.com/parallel-finance/parallel/blob/master/pallets/loans/src/mock.rs#L277